community/working-groups/compliance/content/readme.md

```diff
@@ -0,0 +1 @@
+This folder contains content created by Compliance WG
```

Comment:
I'll take a look at the current warnings we are seeing the linting process. Nonetheless the linting action is failing as this file needs a top level H1.

Reply:
Appreciated